domains/nolan-mai

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "PotBorger",
+        "email": "khanhtrinh232005@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://potborger.github.io/Portfolio/"
```

Comment:
Invalid CNAME, this should just be a hostname without file paths, protocols, URL parameters, etc.